website/src/content/development/notifications.mdx

```diff
@@ -0,0 +1,392 @@
+---
+title: Notifications
+---
+
+In the previous section, we learned how to develop forms. In this section, we'll look at how to provide feedback to users using notifications, especially in response to user actions.
+
+To facilitate the use of notifications, Custom Applications have a built-in notification system.
+
+# Notification system
+
+Notifications are an important means of providing feedback in response to user actions.
+For example, the submission of a form may succeed or fail. The user should be informed about this in an understandable and easy-to-find way.
+
+Notifications are grouped by so-called `domains`. A domain specifies where the notification should be displayed on the page (for instance `page` and `side`).
+
+Various types of notifications are available: `success`, `error`, `warning`, `info`. Depending on the use case you choose the appropriate type. For example, if a form was submitted successfully, you would display a side notification of the type `success`.
+
+Triggering notifications is done using some custom React hooks. Please refer to the [`@commercetools-frontend/actions-global`](/api-reference/commercetools-frontend-actions-global) package for implementation details.
+
+<Info>
+
+For defining the `domain` and the `kind` of a notification, we recommend using the values exposed from `@commercetools-frontend/constants`.
```

Comment:
Also, we should add the `NOTIFICATION_KINDS_*` to the docs page of that package as well.